django-stubs/apps/registry.pyi

```diff
@@ -24,7 +26,7 @@ class Apps:
     def get_app_config(self, app_label: str) -> AppConfig: ...
     # it's not possible to support it in plugin properly now
     def get_models(self, include_auto_created: bool = ..., include_swapped: bool = ...) -> list[type[Model]]: ...
-    def get_model(self, app_label: str, model_name: str | None = ..., require_ready: bool = ...) -> type[Any]: ...
+    def get_model(self, app_label: str, model_name: str | None = ..., require_ready: bool = ...) -> ModelType: ...
```

Comment:
Hi. I don't think it can work like this. The problem is that the model class name is passed in as string to `model_name` argument.

With this construction I suspect it will end up being `type[Any]` as well.

Reply:
This works with the static type checker built into pycharm.

Reply:
After making this change:
![Screenshot_20230112_160729](https://user-images.githubusercontent.com/24929701/212191730-e3ba7c45-7683-4b53-ae92-6bdbf332872c.png)

![Screenshot_20230112_160720](https://user-images.githubusercontent.com/24929701/212191757-a294298c-3946-438c-94bc-ec3ba907ecf0.png)

<hr>

Before:
![Screenshot_20230112_161017](https://user-images.githubusercontent.com/24929701/212191978-e4b138ce-4dec-4f81-a5cc-ae89384ea779.png)